<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>565</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="skinframe">
     <property name="maximumSize">
      <size>
       <width>420</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QGridLayout" name="skinFrameLyt_2">
      <item row="6" column="0">
       <layout class="QVBoxLayout" name="combinedLyt">
        <item>
         <widget class="QPushButton" name="combinedBtn">
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Combined Skinning</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="dualquatnonriging_btn">
          <property name="text">
           <string>Dual Quaternion Non Rigid</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deltascale_btn">
          <property name="text">
           <string>Global Scale to Deltas</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="9" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="offskin_btn">
          <property name="text">
           <string>Turn Off Skins</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="on_skins">
          <property name="text">
           <string>Turn On Skins</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Auto Skin</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="Line" name="line_16">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="mirrorLyt">
           <item>
            <widget class="QLabel" name="mirrorLabel">
             <property name="text">
              <string>Extra Utils</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="mirrorBox">
             <property name="text">
              <string>Mirror</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="meshNameLyt" stretch="0,0,0">
           <item>
            <widget class="QLabel" name="meshNameLoadLabel">
             <property name="text">
              <string>MeshName</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="meshNameLoadField"/>
           </item>
           <item>
            <widget class="QPushButton" name="meshNameLoadBtn">
             <property name="text">
              <string>&lt;&lt;&lt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="0">
          <layout class="QHBoxLayout" name="skinningMethodsLyt">
           <item>
            <layout class="QVBoxLayout" name="softSelectLyt">
             <item>
              <layout class="QHBoxLayout" name="falloffLyt">
               <item>
                <widget class="QLabel" name="falloffLabel">
                 <property name="text">
                  <string>Falloff</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="falloffValueLabel">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSlider" name="falloffSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="softSelectBtn">
               <property name="text">
                <string>SoftSelect Skinning</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_13">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="Line" name="line_14">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="growSelectLyt">
             <item>
              <layout class="QHBoxLayout" name="expandLyt">
               <item>
                <widget class="QLabel" name="expandLabel">
                 <property name="text">
                  <string>Expand</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="expandValueLabel">
                 <property name="text">
                  <string>0</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QSlider" name="expandSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="growSelectBtn">
               <property name="text">
                <string>GrowSelect Skinning</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QVBoxLayout" name="curveOptionsLyt">
           <item>
            <widget class="Line" name="line_15">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="presetsLyt">
             <item>
              <widget class="QLabel" name="curveLabel">
               <property name="text">
                <string>Curve Presets:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="defaultBox">
               <property name="text">
                <string>Curve_Editor</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="strongCenterBox">
               <property name="text">
                <string>Strong_Center </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="bigFallBox">
               <property name="text">
                <string>Big_Falloff</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="customLyt">
             <item>
              <widget class="QCheckBox" name="customBox">
               <property name="text">
                <string>Custom</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="customField"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="4" column="0">
          <layout class="QVBoxLayout" name="curveEditorLyt">
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="save_skin_button">
         <property name="text">
          <string>Save Current Skins</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Show</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="load_button">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="shows_combo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Asset Type</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="assets_type_combo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Asset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="assets_combo"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="version_load_button">
         <property name="text">
          <string>Load Versions</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Version</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="versions_combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="maxCount">
            <number>2147483647</number>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Skin</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="skin_versions_combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Geo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="geos_combo"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="load_skin_button">
         <property name="text">
          <string>Load Skin</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="ref_copy_btn">
           <property name="text">
            <string>Ref-Copy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ref_copy_uv_btn">
           <property name="text">
            <string>Ref-Copy(UV)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="select_bind_jnts">
           <property name="text">
            <string>Select Bind Jnts</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="null_check">
           <property name="text">
            <string>Null</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="copy_skin_to_nurb">
         <property name="text">
          <string>Copy Skin to Nurb</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="click_copy_button">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>1 Click Copy Skin</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
